lib/router/src/lib.rs

```diff
@@ -150,8 +148,8 @@ where
     ///
     /// TODO Attempt to free capacity in the router.
     fn poll_ready(&mut self) -> Poll<(), Self::Error> {
-        // FIXME(kleimkuhler): impl Send for Target and Svc?
-        // tokio::spawn(self.inner._cache_bg);
+        let inner = self.inner.clone();
+        tokio::spawn(inner.cache_bg);
```

Comment:
I don't understand why every `poll_ready` spawns a new `cache_bg`? My expectation is that each router would run a single background task...